src/core/scene/a-scene.js

```diff
@@ -65,11 +65,30 @@ module.exports = registerElement('a-scene', {
           this.setupRenderer();
           this.resize();
         });
+        this.addFullScreenStyles();
         initPostMessageAPI(this);
       },
       writable: true
     },
 
+    addFullScreenStyles: {
+      value: function () {
+        var htmlEl = document.querySelector('html');
+        htmlEl.classList.add('a-html');
+        document.body.classList.add('a-body');
+        this.classList.add('fullscreen');
```

Comment:
I wouldn't call the collision chances minimal. fullscreen is a very generic name, I wouldn't be surprised if a site had a class for that. it's very easy to make the chances zero now. The established pattern is that we prefix things